Source/Neptune.Web/Views/TreatmentBMP/Detail.cshtml

```diff
@@ -43,38 +43,56 @@
             font-size: 75%;
             margin-bottom: 0;
         }
+        .unverify-inventory-btn {
+            font-size: 11px;
+            margin: 0 5px 0 0;
+            text-align: right;
+        }
     </style>
 }
 
 @section RightOfPageTitle{
     @if (ViewDataTyped.CurrentPersonCanManage)
     {
-        <div class="pull-right">
-            @if (ViewDataTyped.TreatmentBMP.InventoryIsVerified)
-            {
-                <span class="helpicon glyphicon glyphicon-ok helpiconGridBlue" style="cursor: default;"></span> 
-                <span style="margin-right: 5px">Inventory has been verified</span>
-            }
-            else
-            {
-                <span class="helpicon glyphicon glyphicon-warning-sign helpiconGridBlue" style="cursor: default;"></span> 
-                <span style="margin-right: 5px">Inventory has not been verified</span>
-            }
+    <div class="pull-right">
+        @if (ViewDataTyped.TreatmentBMP.InventoryIsVerified)
+        {
+            <div style="display:inline-block">
+                <span class="helpicon glyphicon glyphicon-ok helpiconGridBlue" style="cursor: default;"></span>
+                <span style="margin-right: 5px">BMP has been marked as verified</span>
+                
+                @if (ViewDataTyped.CanManageStormwaterJurisdiction)
+                {
+                    @ModalDialogFormHelper.ModalDialogFormLink("Mark BMP Record as Provisional", ViewDataTyped.VerifiedUnverifiedUrl, "Mark BMP Record as Provisional?", 400, "Continue", "Cancel", new List<string> { "help-block", "unverify-inventory-btn" }, null, null)
+                }
+            </div>
+        }
+        else
+        {
+            <div style="display:inline-block">
+                <span class="helpicon glyphicon glyphicon-warning-sign helpiconGridBlue" style="cursor: default;"></span>
+                <span style="margin-right: 5px">BMP has been marked as provisional</span>
```

Comment:
The text to show when the inventory is not verified should still say "Inventory has not been verified"